### ISSUE TITLE
Installer should backup before deleting

### ISSUE THREAD

**Issue by dylanrenwick** [dsisco11/SR_Plugin_Loader#44]
The installer should create a backup before deleting Assembly-Csharp.dll, as if redownloading the file from steam fails for any reason, it renders the game unplayable

**Comment by dsisco11** [comment]
Im fairly sure it does, though it isn't apparent in the code, it should be calling a function that makes a backup before it deletes the original and triggers the steam cache verification.

**Comment by dsisco11** [comment]
Nevermind you are correct. The program currently assumes that if the assembly needs to be "cleaned"(redownloaded) that it is already tainted in some way and useless. However; that's a very bad thing for it to assume.

**Comment by dylanrenwick** [comment]
Yup, I did however notice that a much older version of SRPL (v0.4) did in fact make a backup, which is what gave me the idea in the first place.
